spec/integration/apply_spec.rb

```diff
@@ -84,6 +84,19 @@ def root_config
         resources = result[0]['result']['report']['resource_statuses']
         expect(resources).to include('Notify[hello world]')
       end
+
+      it 'applies the deferred type' do
+        result = run_cli_json(%w[plan run basic::defer] + config_flags)
+        expect(result).not_to include('kind')
+        expect(result[0]['status']).to eq('success')
+        resources = result[0]['result']['report']['resource_statuses']
+
+        local_pid = resources['Notify[local pid]']['events'][0]['desired_value'][/(\d+)/, 1]
```

Comment:
`expect(resources['Notify[local pid]']['events'][0]['desired_value']).to match(/(\d+)/)`

seems clearer.

Reply:
The test is specifically testing that local pid and remote pid are different.